app/controllers/blogs_controller.rb

```diff
@@ -61,10 +60,20 @@ def destroy
     end
   end
 
+  def toggle_status
+      if @blog.draft? 
+        @blog.published! 
+      elsif 
+          @blog.published!
+          @blog.draft? 
```

Comment:
this can be reduced in two lines, like the videos example

Reply:
In a part of the video it is shown this way